### PR TITLE
fix wording with content

### PR DESCRIPTION
**Pull request by fjy**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,7 +15,6 @@
     <p class="lead">Druid is a fast column-oriented distributed data store</p>
     <p><a class="btn btn-lg btn-outline-inverse" href="downloads.html"><span class="fa fa-download"></span> Download</a>
     <a class="btn btn-lg btn-outline-inverse" href="https://github.com/druid-io/druid/"><span class="fa fa-github"></span> GitHub</a></p>
-    <p class="small text-muted">latest stable release {{ site.druid_stable_version }}</p>
```
Comment (gianm) [inline index.html]: why remove this?

File: index.html [hunk 2]
```diff
@@ -85,17 +85,17 @@
...
 
         <span class="fa fa-bar-chart fa"></span><br>
         <h3>Open Source UIs and Rich Client Libraries</h3>
         <p>
-          Try <a href="https://github.com/implydata/pivot">Pivot</a> or <a
-          href="https://github.com/mistercrunch/panoramix">Panoramix</a> for visualizing data.
-          Query Druid using <a href="/docs/{{site.druid_version}}/development/libraries.html">many different languages</a>.
+          Visualize data with <a href="https://github.com/implydata/pivot">Pivot</a> and <a
+          href="https://github.com/mistercrunch/panoramix">Panoramix</a>.
+          Query data in <a href="/docs/{{site.druid_version}}/development/libraries.html">many different languages</a>.
```
Comment (xvrl) [inline index.html]: we might want to point to latest for this one, should we also add grafana?

File: index.html (hunk 1 above)
Comment (fjy) [inline index.html]: it clutters up the front page and the information is repeated on the sidebar

File: index.html (hunk 2 above)
Comment (fjy) [inline index.html]: i dont know if anyone runs grafana in prod

File: index.html (hunk 1 above)
Comment (xvrl) [inline index.html]: the sidebar may not always have the latest release, though

Comment (fjy) [inline index.html]: it should, otherwise we're doing a bad job keeping webpage up to date

Comment (fjy) [inline index.html]: also the download page includes the latest stable version. I don't see what including it so prominent on the front page does

Comment (fjy) [inline index.html]: it is distracting and extra text that isn't needed IMO

Comment (xvrl) [inline index.html]: maybe we can add it to the download button? If we only show the last 4-5 news items, it's possible that the latest release will be hidden

Comment (fjy) [inline index.html]: that'd be cool

Comment (fjy) [inline index.html]: i fixed it

**Comment by xvrl** [comment]
:+1:

**Pull request merged**